libnmstate/nm/checkpoint.py

```diff
@@ -124,6 +126,21 @@ def create(self):
                 raise NMCheckPointPermissionError(str(e))
             raise NMCheckPointCreationError(str(e))
 
+        GLib.timeout_add(
+            self._timeout * 0.5, self._refresh_checkpoint_timeout, None,
+        )
+
+    def _refresh_checkpoint_timeout(self, data):
+        cancellable, cb, cb_data = (None, None, None)
+        nm_client = client()
+        if nm_client:
+            nm_client.checkpoint_adjust_rollback_timeout(
+                self._dbuspath, self._timeout, cancellable, cb, cb_data
+            )
+            return True
+        else:
+            return False
```

Comment:
nit: Can we use G_SOURCE_CONTINUE and G_SOURCE_REMOVE ?

Reply:
Should be `GLib.SOURCE_CONTINUE` in python

Reply:
Done, thanks!